.github/actions/build-wheels/action.yml

```diff
@@ -0,0 +1,73 @@
+name: 'Build Wheels'
+description: 'Build wheels for ofrak'
+inputs:
+  python-version:
+    description: 'Python version to build for'
+    required: true
+  os:
+    description: 'Operating system'
+    required: true
+runs:
+  using: 'composite'
+  steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+        lfs: false
+
+    - name: Setup LFS cache
+      uses: ./.github/actions/lfs-cache
+
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: ${{ inputs.python-version }}
+        cache: pip
+        cache-dependency-path: |
+          requirements-*.txt
+          **/setup.py
+
+    - name: Set up Node.js
+      uses: actions/setup-node@v4
+      with:
+        node-version: '18'
+
+    - name: Build frontend
+      shell: bash -euxo pipefail {0}
+      run: make -C ofrak_core src/ofrak/gui/public
+
+    - name: Install build tools
+      shell: bash -euxo pipefail {0}
+      run: |
+        python -m pip install -r requirements-pip.txt
+        python -m pip install -r requirements-build.txt
+
+    - name: Set Python version for cibuildwheel
+      id: py-version
+      shell: bash -euxo pipefail {0}
+      run: |
+        PY_VERSION="${{ inputs.python-version }}"
+        PY_VERSION_NO_DOT=${PY_VERSION//./}
+        echo "version=${PY_VERSION_NO_DOT}" >> $GITHUB_OUTPUT
```

Comment:
Why make this a separate step? This could probably be folded into the step below and avoid passing variables between workflow steps.